packages/fx-core/resource/package.nls.json

```diff
@@ -1175,5 +1175,9 @@
   "driver.botFramework.summary.create": "The bot registration has been created successfully (%s).",
   "driver.botFramework.summary.update": "The bot registration has been updated successfully (%s).",
   "driver.botFramework.progressBar.title": "botFramework/createOrUpdateBot",
-  "driver.botFramework.progressBar.createOrUpdateBot": "Creating or updating bot registration."
+  "driver.botFramework.progressBar.createOrUpdateBot": "Creating or updating bot registration.",
+  "driver.generateAppsettings.description": "Add or update Teamsfx related parameters appsettings.Development.json.",
```

Comment:
Add or update Teamsfx related parameters **to appsettings file**

Since we do not limited to only `appsettings.Development.json`

Reply:
Done.